### PR TITLE
Modal Background & box shadow fixes

### PR DESCRIPTION
**Pull request by Bonapara**
# Before

CleanShot 2024-11-14 at 19 20 59

# After

CleanShot 2024-11-14 at 19 21 34

### CODE REVIEW

File: packages/twenty-front/src/modules/ui/layout/modal/components/Modal.tsx [hunk 1]
```diff
@@ -15,9 +15,15 @@ const StyledModalDiv = styled(motion.div)<{
   size?: ModalSize;
   padding?: ModalPadding;
   isMobile: boolean;
+  modalVariant: ModalVariants;
 }>`
   display: flex;
   flex-direction: column;
+  box-shadow: ${({ theme }) => theme.boxShadow.superHeavy};
```
Comment (Weiko) [inline packages/twenty-front/src/modules/ui/layout/modal/components/Modal.tsx:22]: You probably forgot to remove that line after implementing the variant below 🙂

**Review by Weiko — COMMENTED** [review]
Left a comment otherwise LGTM!

File: packages/twenty-front/src/modules/ui/layout/modal/components/Modal.tsx (hunk 1 above)
Comment (Bonapara) [reply packages/twenty-front/src/modules/ui/layout/modal/components/Modal.tsx]: Thanks @Weiko!

**Review by Weiko — APPROVED** [review]
LGTM!

**Pull request merged**